website/src/app/page.tsx

```diff
@@ -44,114 +44,69 @@ export const metadata: Metadata = {
-                  <Link
-                    href="https://app.firezone.dev/sign_up"
-                    className="group inline-flex items-center py-0.5 text-neutral-200 text-lg font-semibold border-b-2 border-neutral-200 hover:border-primary-450 hover:text-primary-450 transition transform duration-50"
-                  >
-                    Sign up now
-                    <HiArrowLongRight className="group-hover:translate-x-1 group-hover:scale-110 duration-50 transition transform ml-2 -mr-1 w-7 h-7" />
-                  </Link>
-                </div>
-                <div className="my-4 flex items-center">
-                  <button
-                    type="button"
-                    className="group shadow-lg shadow-primary-700 text-lg w-48 inline-flex shadow-lg justify-center items-center py-3 px-5 font-semibold text-center text-white rounded bg-primary-450 hover:ring-1 hover:ring-primary-450 duration-50 transform transition"
-                  >
-                    <Link href="/contact/sales">Book a demo</Link>
-                    <HiArrowLongRight className="group-hover:translate-x-1 group-hover:scale-110 duration-50 transition transform ml-2 -mr-1 w-7 h-7" />
-                  </button>
-                </div>
-              </div>
+            <div className="mt-8 md:mt-0 flex items-center">
+              <button
+                type="button"
+                className="group shadow-lg shadow-primary-700 text-lg w-48 inline-flex shadow-lg justify-center items-center py-3 px-5 font-semibold text-center text-white rounded bg-primary-450 hover:ring-1 hover:ring-primary-450 duration-50 transform transition"
+              >
+                <Link href="/contact/sales">Book a demo</Link>
+                <HiArrowLongRight className="group-hover:translate-x-1 group-hover:scale-110 duration-50 transition transform ml-2 -mr-1 w-7 h-7" />
+              </button>
             </div>
           </div>
-          <div className="flex items-center justify-center px-4 md:px-0">
-            <video
-              className="shadow-lg rounded"
-              width="960"
-              height="540"
-              loop
-              autoPlay
-              playsInline
-              muted
-            >
-              <source src="/images/overview-screencap.webm" type="video/webm" />
-              Your browser does not support the video tag with WebM-formatted
-              videos.
-            </video>
+        </div>
+        <div className="mt-12 max-w-screen-md mx-auto">
+          <div className="text-sm mb-4 flex justify-center font-light text-neutral-500">
+            Backed by{" "}
+            <Image
+              src="/images/yc-logo-gray.svg"
+              alt="yc logo gray"
+              width={100}
+              height={40}
+              className="mx-1.5"
+            />{" "}
+            and trusted by hundreds of organizations
           </div>
+          <CustomerLogosGrayscale />
```

Comment:
Add bigger padding-y by at least 24px, add dark gradient to left and right sides to make it appear fading out